pkg/convert/json_ngxtranslate_test.go

```diff
@@ -0,0 +1,129 @@
+package convert
+
+import (
+	"testing"
+
+	"github.com/expect-digital/translate/pkg/model"
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_fromNgxTranslate(t *testing.T) {
```

Comment:
I'm not sure, but should we commit tests also? I remember that @janishorsts said that its not needed

Reply:
It was complicated function with recursion and I needed to make sure that it does the thing I want it to do. So I made simple tests